docs/freeform_trees.rst

```diff
@@ -0,0 +1,23 @@
+:tocdepth: 3
+
+.. _freeform_trees:
+
+
+Freeform translation tree
+*************************
+Before you can start using freeform trees, please be sure you followed all the steps in: :ref:`getting_started`.
+
+If you specified the `WAGTAILTRANS_SYNC_TREE` in your settings as `False` you will be using the freeform trees.
+the freeform trees are used if you do not want to keep your language trees the same for all languages.
```

Comment:
@jjanssen Good point. We could do that in the index.rst.. I'll try to add that asap.